plugins/keyboard-navigation/package.json

```diff
@@ -41,15 +41,15 @@
   "devDependencies": {
     "@blockly/dev-scripts": "^2.0.1",
     "@blockly/dev-tools": "^7.0.2",
-    "blockly": "^10.0.0",
+    "blockly": "^10.2.0-beta.0",
```

Comment:
Looks like new API is expected in 10.2.  Perhaps update the bug, which says it will be in 11?

Reply:
I'll just change it to eoq3 since we don't know what the version is going to be.